databuilder/query_engines/base_sql.py

```diff
@@ -212,7 +214,15 @@ def get_sql_subtract(self, node):
     def get_sql_string_contains(self, node):
         haystack = self.get_expr(node.lhs)
         needle = self.get_expr(node.rhs)
-        return haystack.contains(needle)
+        escaped_needle = self.string_replace(needle, "/", "//")
+        escaped_needle = self.string_replace(escaped_needle, "%", "/%")
+        escaped_needle = self.string_replace(escaped_needle, "_", "/_")
+        return haystack.contains(escaped_needle, escape="/")
```

Comment:
If we've got a choice of escape characters, why not backlash?

Reply:
`/` is the SQLAlchemy default. I thought that was odd too and tried using a backslash and then the tests started failing on Spark, and I didn't look much further into it. Worth a comment, do you think?

Reply:
No, it's purely internal. I'll add a comment though.